cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsCredentialConnector.java

```diff
@@ -168,6 +184,29 @@ private CloudCredentialStatus verifyAccessKeySecretKeyIsAssumable(CloudCredentia
         return credentialStatus;
     }
 
+    private void checkRoleIsAssumableWithoutExternalId(CredentialVerificationContext credentialVerificationContext, AwsCredentialView awsCredential) {
+        if (credentialVerificationContext.getCreationVerification()) {
+            String roleArn = awsCredential.getRoleArn();
+            try {
+                credentialClient.retrieveSessionCredentialsWithoutExternalId(awsCredential);
+                String message = String.format("CDP Control Pane is able to perform 'sts:AssumeRole' on '%s' role without external id. " +
```

Comment:
I am not sure but should we somehow show what is the external id which is missing ? I guess not but lets check